src/RecoveryServices/RecoveryServices.Backup/RecoveryServices.Backup.csproj

```diff
@@ -12,6 +12,7 @@
   </PropertyGroup>
 
   <ItemGroup>	  
+	<ProjectReference Include="..\..\Compute\Compute\Compute.csproj" />	  
```

Comment:
Please don't reference csproject of other modules in your non-test csproject. You can refer package directly